src/main/scala/org/scalameter/Parameters.scala

```diff
@@ -1,20 +1,31 @@
 package org.scalameter
 
+import scala.language.existentials
 import scala.collection.{Iterable, immutable}
+import Parameters.P
+
 
 @SerialVersionUID(4203959258570851398L)
-case class Parameters(axisData: immutable.ListMap[String, Any]) {
+case class Parameters(axisData: immutable.ListMap[Parameter[P], P]) {
```

Comment:
It is unconventional to use single letter abstract type names for concrete types. See my comment about existentials below.